security/antispam/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=          antispam
+PLUGIN_VERSION=       0.1
+PLUGIN_COMMENT=       Package to configure postfix, rspamd and clamav to protect your network from Malware and Spam
+PLUGIN_DEPENDS=       rspamd os-clamav postfix-sasl
```

Comment:
er, please, no, we need a clean approach that works with each plugin: rspamd, (we alread have clamav), postfix